kahuna/public/js/preview/image.html

```diff
@@ -48,13 +48,13 @@
         <div class="flex-container">
             <ul class="bottom-bar__meta-item preview__collections">
                 <li class="preview__collections__collection"
-                    ng:repeat="collection in ctrl.image.data.collections"
-                    gr-tooltip="Click to open collection: {{collection.data.path.join(' ▸ ')}}"
+                    ng:repeat="collection in ctrl.image.data.collections track by collection.data.pathId"
```

Comment:
It's recommended to always use `track by` in ngRepeat so that Angular can recycle nodes for items uniquely identified by their `collection.data.pathId`. In this instance I think it would avoid re-rendering all the collections if the list was modified (since without this, nodes would be identified by the object identity of each collection, which is entirely replaced when collections are added/removed).

I don't expect a big impact of this but I think it's good practice to use with all ngRepeat.